src/backend/el.d

```diff
@@ -0,0 +1,221 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1985-1995 by Symantec
+ *              Copyright (c) 1999-2016 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     backendlicense.txt
+ * Source:      $(DMDSRC backend/_el.d)
+ */
+
+module ddmd.backend.el;
+
+import ddmd.backend.cdef;
+import ddmd.backend.cc;
+import ddmd.backend.type;
+
+import ddmd.backend.cc : Symbol;
```

Comment:
Because of the double definition problem you noted earlier. This produces an alias which says which one is needed. It's a hack to get it to compiler. A proper fix is not so practical until all the .h files are translated, for reasons already mentioned.